securedrop_client/logic.py

```diff
@@ -679,39 +641,34 @@ def export_file_to_usb_drive(self, file_uuid: str, passphrase: str) -> None:
         is_downloaded is set to False.
         '''
         file = self.get_file(file_uuid)
-        logger.info('Exporting file {}'.format(file.original_filename))
```

Comment:
looks like you accidentally removed a log line

Reply:
Yep.

Reply:
just a nit but i think we should add this log line back